templates/frameworks/list.html.twig

```diff
@@ -113,98 +111,37 @@
-                        {#</fieldset>#}
-                        {#</div>#}
-
-
-
-
-
-
-
-
-
-
-
-                        {#<div class="govuk-form-group govuk-form-group--enclosure">#}
-                        {#<fieldset class="govuk-fieldset">#}
-                        {#<legend class="govuk-fieldset__legend govuk-fieldset__legend--m">#}
-                        {#<h1 class="govuk-fieldset__heading">#}
-                        {#Results per page#}
-                        {#</h1>#}
-                        {#</legend>#}
-                        {#<span id="waste-hint" class="govuk-hint">#}
-                        {#10 results per page#}
-                        {#</span>#}
-                        {#<div class="govuk-radios govuk-radios--small">#}
-                        {#<div class="govuk-radios__item govuk-radios__item--small">#}
-                        {#<input class="govuk-radios__input govuk-radios__input--small" id="results-per-page-1" name="where-do-you-live" type="radio" value="england">#}
-                        {#<label class="govuk-label govuk-radios__label" for="results-per-page-1">#}
-                        {#10#}
-                        {#</label>#}
-                        {#</div>#}
-                        {#<div class="govuk-radios__item govuk-radios__item--small">#}
-                        {#<input class="govuk-radios__input govuk-radios__input--small" id="results-per-page-2" name="where-do-you-live" type="radio" value="scotland">#}
-                        {#<label class="govuk-label govuk-radios__label" for="results-per-page-2">#}
-                        {#25#}
-                        {#</label>#}
-                        {#</div>#}
-                        {#<div class="govuk-radios__item govuk-radios__item--small">#}
-                        {#<input class="govuk-radios__input govuk-radios__input--small" id="results-per-page-3" name="where-do-you-live" type="radio" value="wales">#}
-                        {#<label class="govuk-label govuk-radios__label" for="results-per-page-3">#}
-                        {#50#}
-                        {#</label>#}
-                        {#</div>#}
-                        {#<div class="govuk-radios__item govuk-radios__item--small">#}
-                        {#<input class="govuk-radios__input govuk-radios__input--small" id="results-per-page-4" name="where-do-you-live" type="radio" value="northern-ireland">#}
-                        {#<label class="govuk-label govuk-radios__label" for="results-per-page-4">#}
-                        {#View all#}
-                        {#</label>#}
-                        {#</div>#}
-                        {#</div>#}
-                        {#</fieldset>#}
-                        {#</div>#}
-
-
                     </form>
 
                 </div>
-                <div class="govuk-grid-column-two-thirds">
-
+                {% if guided_match_flag is defined %}
+                    {% if query is defined and query in keywords%}
```

Comment:
I would suggest combining these into a single if (with 3 conditions), so that the nested else case can be removed. 
If the feature is requested but the query doesn't match, we simply go into the 'regular' else mode below.